kooplex-proxy/proxy/main.py

```diff
@@ -27,42 +27,42 @@ def read_root():
 
 
 @app.get("/country_samples/", response_model=List[schemas.MView_CountrySamples])
-def read_country_samples(skip: int = 0, limit: int = 100):
-    country_samples = crud.get_country_samples(next(get_db()), skip=skip, limit=limit)
+def read_country_samples(skip: int = 0, limit: int = 100, schema='sandbox_public'):
```

Comment:
It is not a good idea to define a default schema name in functions. The schema names may likely different between the staging database and production database or from release to release.